app/res/layout/menu_list_item_modern.xml

```diff
@@ -1,28 +1,30 @@
 <?xml version="1.0" encoding="utf-8"?>
 <org.commcare.views.ShrinkingLinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-                                            xmlns:app="http://schemas.android.com/apk/res-auto"
-              android:layout_width="fill_parent"
-              android:layout_height="fill_parent"
-              app:shrinkable_view="@+id/row_txt"
-              android:textAppearance="@style/EntityItemText">
+                                          xmlns:app="http://schemas.android.com/apk/res-auto"
+                                          android:layout_width="fill_parent"
+                                          android:layout_height="fill_parent"
+                                          android:clipChildren="false"
+                                          android:textAppearance="@style/EntityItemText"
+                                          app:shrinkable_view="@+id/row_txt">
 
     <ImageView
         android:id="@+id/row_img"
-        android:layout_width="wrap_content"
-        style="@style/ListContentV2"/>
+        style="@style/ListContentV2"
+        android:layout_width="wrap_content"/>
 
     <TextView
         android:id="@+id/row_txt"
+        style="@style/ListContentV2"
         android:layout_width="wrap_content"
         android:layout_weight="1"
-        style="@style/ListContentV2"
         android:ellipsize="end"
         android:maxLines="2"/>
 
-    <org.commcare.views.media.AudioButton
+    <org.commcare.views.media.AudioPlaybackButton
         android:id="@+id/row_soundicon"
+        android:clipChildren="false"
```

Comment:
oops, but in this case it isn't needed :) just an artifact of me blindly trying to make UI things work on Android.